### PR TITLE
fixed support links

### PR DESCRIPTION
**Pull request by MichaelRupertDev**
Fixed support links as seen in a comment here: https://github.com/borgbackup/borg/pull/3361

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -141,7 +141,7 @@ Links
 * `Web-Chat (IRC) <http://webchat.freenode.net/?randomnick=1&channels=%23borgbackup&uio=MTY9dHJ1ZSY5PXRydWUa8>`_ and
   `Mailing List <https://mail.python.org/mailman/listinfo/borgbackup>`_
 * `License <https://borgbackup.readthedocs.org/en/stable/authors.html#license>`_
-* `Security contact <https://borgbackup.readthedocs.io/en/latest/support.html#security-contact>`_
+* `Security contact <https://borgbackup.readthedocs.io/en/latest/free.html#security-contact>`_
```
Comment (ThomasWaldmann) [inline README.rst]: guess this should stay?
the file in the docs is still named support.html, isn't it?
just the file on the homepage is free.html.

Comment (MichaelRupertDev) [reply README.rst]: That is correct. I searched for all usages of support.html, and those are the only ones still around.

Comment (ThomasWaldmann) [reply README.rst]: I mean the html docs file name is still support.html (so do not change that to free.html).

**Comment by ThomasWaldmann** [comment]
thanks! hopefully, it's all in sync now. :)

**Pull request merged**